### PR TITLE
Installation instructions for extensions

### PR DESCRIPTION
**Pull request by tibu**

### CODE REVIEW

File: readme/clipper.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ The Web Clipper is a browser extension that allows you to save web pages and scr
 
 <img src="https://joplinapp.org/images/WebExtensionScreenshot.png" style="max-width: 50%; border: 1px solid gray;">
 
+# Installing the extension
+
+## In Chrome
+
+- Open https://chrome.google.com/webstore/detail/joplin-web-clipper/alofnhikmmkdbbbgpnglcpdollgjjfek
```
Comment (tessus) [inline readme/clipper.md:11]: This link is not valid and results in a 404.

**Review by tessus — COMMENTED** [review]
We already have install instructions on the main page. I'm not sure, if we sho8uld duplicate that info.

File: readme/clipper.md (hunk 1 above)
Comment (roman-r-m) [reply readme/clipper.md:11]: Strange, it loads ok for me

Comment (tessus) [reply readme/clipper.md:11]: Oops. My bad. I missed the line break.

Comment (tessus) [reply readme/clipper.md:11]: @tibu please ignore the comment about the 404.

However, I'm still not sure, if we should duplicate info. I'm not a fan of doing so. We duplicate info in the terminal.md file. It has already happened that we update one, but not the other.

**Comment by tessus** [comment]
We duplicate info in the terminal.md file. It has already happened that we update one, but not the other.

Laurent, what do you think?

**Comment by tibu** [comment]
I don't think there are installation instructions on  the main page. At
least I did not find them.

Can you please post a screenshot about the 404 and browser / os type too.
I'll try to debug it.

Helmut K. C. Tessarek ***@***.***> ezt írta (időpont: 2021.
márc. 28., V 19:13):

**Comment by tessus** [comment]
> Can you please post a screenshot about the 404 and browser / os type too.
I'll try to debug it.

The diff did a line break I missed, so I only copied half of the URL for testing. Please ignore.

> I don't think there are installation instructions on  the main page. At
least I did not find them.

image

Yes, you are right. It's very easy to miss. Maybe we should add it to the web cliipper file after all.
I really had the impression we had a similar table we have for desktop and mobile, but I must have dreamt that.

**Comment by tibu** [comment]
Shall I add the table to the main page? And then I can just simply
reference from here the main page with a direct link. (or the other way.
Table here, links there to this section)

Helmut K. C. Tessarek ***@***.***> ezt írta (időpont: 2021.
márc. 28., V 19:46):

**Comment by tessus** [comment]
Since we have all download links on the main page, I think we should do the same for clipper. 

In the `Web Clipper` section a table like:

image

But instead of `Operating System` it should say `Browser`, and I'm sure there are similar badges for Firefox and Chrome somewhere.

The current text can be then changed to: _The Web Clipper is a browser extension that allows you to save web pages and screenshots from your browser. For more information on how to use it, see the Web Clipper Help Page._

Then in the web clipper page you can have an installation section and just reference the webclipper section of the main page. Unless Laurent is ok with duplicating the table into the web clipper page. Personally I try to avoid such things.

**Comment by laurent22** [comment]
Hmm, I don't know, I purposely didn't add the links to this page because I want users to install it via the application. That way, they can follow the steps including the most important one: Enable the Web Clipper service.

Otherwise they might install the extension directly from that link, then wonder why it doesn't work.

**Comment by tessus** [comment]
Ah, I see. Yes, this makes sense, but people still ask how to install the Web Clipper and where to download the extensions.
Maybe in this case we should add to the main page that they have to go to `Preferences > Web Clipper` or something.

If we had deep linking, we could jus add a link `joplin://settings/webclipper` ;-) I really have to look into how to do that when I have more time...

**Comment by tessus** [comment]
@tibu did you see Laurent's comment? I think we can close this, since the first sentence in the web clipper help already tells users to open the config and follow the instructions.

The links are in that config section so I'm not sure, if we really need to add anything.

**Pull request closed without merging**